docs/contribute.md

```diff
@@ -331,7 +331,7 @@ documentation.
 Verify that the code and documentation look as expected:
 
 - Learn about the documentation tools [here](https://docs.flyte.org/en/latest/community/contribute.html#documentation)
-- Install the requirements by running `pip install -r docs-requirements.txt`.
+- Install the requirements by running `pip install -r docs-requirements.in`.
```

Comment:
why do we change it?

Reply:
When I am using python3.9 to develop, use `pip install -r docs-requirements.txt` will have the error below.
However, use `pip install -r docs-requirements.in` can solve the case. 
<img width="801" alt="image" src="https://github.com/flyteorg/flytesnacks/assets/76461262/c4ed7a20-46cc-4e58-9aa4-9683945c0a2e">

Reply:
Would this be resolved by running `pip-compile docs-requirements.in` first, then `pip install -r requirements.txt`?

Reply:
In my case, I use `python 3.9` and the error says that some modules required `python>=3.10`.
So I just install `docs-requirements.in ` instead.

Do you mean
1. `pip-compile docs-requirements.in`
2. `pip install -r docs-requirements.txt`
compile it again, and install it?
I think it might work, too, but installing `docs-requirements.in` will give us more flexibility to support all python versions, at least from `python3.8` to `python3.12`.

Reply:
I think we will want to resolve this in a separate PR so it doesn't block merging the ChatGPT example and updates to the other agent examples. @Future-Outlier would you be able to undo this change in this branch and create a separate branch and PR for it?

Reply:
No problem, thank you！